python/ray/data/tests/test_execution_optimizer.py

```diff
@@ -1078,6 +1079,18 @@ def test_from_torch_e2e(ray_start_regular_shared, enable_optimizer, tmp_path):
     _check_usage_record(["FromItems"])
 
 
+def test__blocks_to_input_buffer_op_name(
+    ray_start_regular_shared,
+    enable_optimizer,
```

Comment:
we don't need `enable_optimizer`, right?

Reply:
yeah you're right, removed. thanks